css/index.css

```diff
@@ -1 +1,369 @@
\ No newline at end of file
+
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+
+
+	/* ===== About Page =======*/
+
+/* Makes Everything Centered */
+
+html {
+    font-size: 62.5%;
+}
+
+h2 {
+    font-weight: bold;
+    font-size: 2.4rem;
+}
+
+
+p, address, footer {
+    font-size: 1.6rem;
+}
+
+p {
+    line-height: 1.25;
+}
+
+
+
+.container {
+    /* width: 90%  */
+    max-width: 880px; 
+	margin: 0 auto; 
+    overflow: hidden;
+ 	
+ }
+
+
+  /* Header, Nav, and Logo */
+
+header {
+    margin-top: 30px;
+   
+}
+
+
+header nav ul {
+    display: flex;
+    align-items: center;
+    
+}
+
+
+header nav ul .item {
+    flex: 1;
+}
+
+header nav ul .item a {
+    text-decoration: none;
+    color: gray;
+    font-size: 1.6rem;
+}
+
+header nav ul .item a:hover {
+    font-weight: bold;
+    color: black;
+}
+
+
+
+
+
+	 /* Main and  First Image */
+     
+     .main {
+         display: flex;
+         margin-top: 30px;
+    
+     }
+
+     .main .inner1, .inner2 {
+         flex: 1;
+     }
+
+     
+     .main .inner1 .inno {
+         display: flex;
+         flex-direction: column;
+         justify-content: center;
+         align-items: center;
+         height: 400px;
+         text-align: center;
+     }
+
+     .main .inner1 .inno h1 {
+         font-size: 8.0rem;
+         text-align: center;
+     }
+
+     button {
+         border: 1px solid black;
+         font-size: 1.6rem;
+         background: white;
+         width: 45%;
+         height: 32px;
+         margin-top: 10px;
+         cursor: pointer;
+         transition: 250ms ease-in-out;
+     }
+
```

Comment:
I like the button animation! People are usually super impressed by even simple animation. So it's worthwhile to have a few tricks like this up your sleeve. Keep playing around with it!